Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-createSearchIndex.txt

MongoDB\\Collection::createSearchIndex()
========================================

.. meta::

>    :description: Create an Atlas Search or Vector Search index for a collection using `MongoDB\Collection::createSearchIndex()` in PHP, specifying index type and options.

Reviewer: ```suggestion
   :description: Create an Atlas Search or Vector Search index for a collection with the MongoDB PHP Library, specifying index type and options.
```
